cl/lib/utils.py

```diff
@@ -118,3 +118,75 @@ def human_sort(
         sorter = lambda item: [convert(c) for c in re.split("([0-9]+)", item)]
 
     return sorted(unordered_list, key=sorter)
+
+
+def microservice(
+    service: str,
+    method: str = "POST",
+    doc: Any = None,
```

Comment:
Since we deal with so many documents and the term is so loaded, how about calling it `item` instead? 

I think you can also type this as a Model object, since all our object inherit from that, but I'm not sure. I think it'd be a helpful hint if it worked though.

Reply:
neat.